scripts/bundleSize.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Size limit for all bundles used by each service (K)
+limit=730
```

Comment:
Add a minimum too, it's too easy for us to forget these values, if we can set the range and regularly update it it'll be much better forced awareness of bundle size changes for all devs.

Reply:
Done